packages/vega-typings/types/spec/projection.d.ts

```diff
@@ -22,45 +22,63 @@ export type ProjectionType =
   | 'transverseMercator';
 export interface BaseProjection {
   /*
-   * The type of the projection. The default is `"mercator"`.
+   * The cartographic projection to use. This value is case-insensitive, for example `"albers"` and `"Albers"` indicate the same projection type.
+   *
+   * __Default value:__ `mercator`
    */
   type?: ProjectionType | SignalRef;
-  /*
-   * The clip angle of the projection.
+
+  /**
+   * The projection’s clipping circle radius to the specified angle in degrees. If `null`, switches to [antimeridian](http://bl.ocks.org/mbostock/3788999) cutting rather than small-circle clipping.
```

Comment:
```suggestion
   * The projection's clipping circle radius to the specified angle in degrees. If `null`, switches to [antimeridian](http://bl.ocks.org/mbostock/3788999) cutting rather than small-circle clipping.
```